.changeset/silly-bulldogs-complain.md

```diff
@@ -0,0 +1,24 @@
+---
+'@openfn/language-dhis2': minor
+---
+
+- Implement a namespaced `http` for all generic functions.
+- Add a generic request function in the `http` file.
+- Remove all callbacks
```

Comment:
This was a surprise - I think you should have flagged this in the PR title

You did mention it in the description, so that's good. I missed it, which is my fault, but still. I'd like to see a big change like this reflected in the PR title. Because if I one day want to look through history and work out where callbacks were removed, the title would help me out a lot.

A good rule of thumb: major changes should be reflected in PR titles :)